bb/viewer_resources_hashing.clj

```diff
@@ -51,8 +51,10 @@
 (defn lookup-url [lookup-hash]
   (str gs-bucket "/lookup/" lookup-hash))
 
-(defn cas-link [hash]
-  (str base-url "/data/" hash))
+(defn asset-name [hash suffix]
+  (str "/data/" hash
```

Comment:
Let's maybe keep the things in `/data` content-addressed and move this to `/assets`.